aws/data_source_aws_route.go

```diff
@@ -190,3 +191,13 @@ func getRoutes(table *ec2.RouteTable, d *schema.ResourceData) []*ec2.Route {
 	}
 	return routes
 }
+
```

Comment:
Moved here temporarily. Will need to clean up when support for managed prefix list IDs added.